scripts/commands/checkLinks.ts

```diff
@@ -179,6 +189,7 @@ async function determineCurrentDocsFileBatch(
       .join(".");
 
     toCheck.push(`docs/api/qiskit/release-notes/${currentVersion}.md`);
+    toLoad.push("docs/api/qiskit/release-notes/0.{45,46}.md");
```

Comment:
We need the 0.44 release notes to fix the last broken link. I added the comment to clarify that we only need them as long as qiskit 1.0 is the latest version. After that, we'll need to move it to the historical `toLoad`
```suggestion
    // Necessary files for docs/api/qiskit/release-notes/1.0.md
    toLoad.push("docs/api/qiskit/release-notes/0.{44,45,46}.md");
```